### PR TITLE
Internship final countdown (TU TUTUTU TU)

### PR DESCRIPTION
**Pull request by Porow**

### CODE REVIEW

File: Users/anroche.go [hunk 1]
```diff
@@ -28,3 +28,37 @@ func Anroche(option string, event *Struct.Message) bool {
...
+
+	ts += +Decisecond / 2
+	d := (ts / Day)
+	ts = ts % Day
+	h := ts / time.Hour
+	ts = ts % time.Hour
+	m := ts / time.Minute
+	ts = ts % time.Minute
+	s := ts / time.Second
+	event.API.PostMessage(event.Channel, fmt.Sprintf("Libération dans %02d jours, %02d heures %02d minutes %02d secondes, d, h, m, s), Struct.SlackParams)
```
Comment (eleboucher) [inline Users/anroche.go]: quote not closed

File: Users/anroche.go [hunk 2]
```diff
@@ -28,3 +28,37 @@ func Anroche(option string, event *Struct.Message) bool {
 	event.API.PostMessage(event.Channel, fmt.Sprintf("Fin du stage dans %02d jours, %02d heures %02d minutes %02d secondes :e2r5:", d, h, m, s), Struct.SlackParams)
 	return true
 }
+
+func Calba(option string, event *Struct.Message) bool {
```
Comment (eleboucher) [inline Users/anroche.go]: better to add it into a new file

File: Users/anroche.go [hunk 3]
```diff
@@ -28,3 +28,37 @@ func Anroche(option string, event *Struct.Message) bool {
...
+	h := ts / time.Hour
+	ts = ts % time.Hour
+	m := ts / time.Minute
+	ts = ts % time.Minute
+	s := ts / time.Second
+	event.API.PostMessage(event.Channel, fmt.Sprintf("Libération dans %02d jours, %02d heures %02d minutes %02d secondes, d, h, m, s), Struct.SlackParams)
+	return true
+}
+
+func Fciprian(option string, event *Struct.Message) bool {
```
Comment (eleboucher) [inline Users/anroche.go]: add it into a new file

**Review by eleboucher — CHANGES_REQUESTED** [review]

File: Users/anroche.go (hunk 3 above)
Comment (Porow) [reply Users/anroche.go]: will do

**Comment by Porow** [comment]
Updated, please check for duplicates and remove it if not needed.

**Comment by eleboucher** [comment]
c' est tres moche comme PR

**Review by eleboucher — APPROVED** [review]

**Pull request merged**